virt/virt-2-4-release-notes.adoc

```diff
@@ -155,37 +200,19 @@ value by running `oc describe node <node>` for all nodes and looking at the
 `cpu-model-<name>` labels. Select the CPU model that is present on all of your
 nodes.
 
-* When running `virtctl image-upload` to upload large VM disk images in `qcow2`
-format, an end-of-file (EOF) error may be reported after the data is
-transmitted, even though the upload is either progressing normally or completed.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1789093[*BZ#1789093*])
-+
-Run the following command to check the status of an upload on a given PVC:
-+
-----
-$ oc describe pvc <pvc-name> | grep cdi.kubevirt.io/storage.pod.phase
-----
-
 * When attempting to create and launch a virtual machine using a Haswell CPU,
 the launch of the virtual machine can fail due to incorrectly labeled nodes.
-This is a change in behavior from previous versions of {VirtProductName},
-where virtual machines could be successfully launched on Haswell hosts.
+This is a change in behavior from previous versions of container-native
+virtualization, where virtual machines could be successfully launched on Haswell hosts.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1781497[*BZ#1781497*])
 +
 As a workaround, select a different CPU model, if possible.
 
-* If you select a directory that shares space with your operating system,
-you can potentially exhaust the space on the partition, causing the node to
-be non-functional. Instead, create a separate partition
-and point the hostpath provisioner to that partition so it will not
-interfere with your operating system.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1793132[*BZ#1793132*])
```

Comment:
Just making sure that this was removed on purpose (note above says this section is unchanged)

Reply:
Looks like the text got dropped. Added it back.